src/libNetwork/P2PComm.h

```diff
@@ -98,31 +97,35 @@ class P2PComm
     using Dispatcher
         = std::function<void(const std::vector<unsigned char>&, const Peer&)>;
 
+    using Broadcast_list_func = std::function<std::vector<Peer>(
```

Comment:
Not sure if we are enforcing the naming convention or not, but `BraodcastListFunc` is preferred in google's coding [guide](https://google.github.io/styleguide/cppguide.html#Type_Names). @jiayaoqijia, any idea?

Reply:
It's good guideline and I will modify this

Reply:
Agree

Reply:
Since we use clang-format, we can choose a conversion it support. You can take a look at https://clang.llvm.org/docs/ClangFormatStyleOptions.html#configurable-format-style-options.